.gitignore

```diff
@@ -1,6 +1,17 @@
-.Rproj.user
+#Targets directry
+_targets
+
+#All pipeline output
+*/out/*
+
+#R files
+# History files
 .Rhistory
+.Rapp.history
+
+# Session Data files
 .RData
-.Ruserdata
 
-*.Rproj
```

Comment:
@lekoenig I see that you have removed the `*.Rproj` file from the `.gitignore`. However, when I pulled this branch locally I did not see `ds-pipelines-targets-example-wqp.Rproj`. 

Can you add another commit that adds this file back in?